lib/game.dart

```diff
@@ -813,11 +813,11 @@ class _SudokuGameState extends State<SudokuGame> with TickerProviderStateMixin {
 
           // also add it to our local list
           DateTime now = DateTime.now();
-          scores.add(Score(_stopwatch.elapsed + _stopwatchOffset, "${now.day} ${DateFormat.yMMM().format(now)}"));
+          scores.add(0, Score(_stopwatch.elapsed + _stopwatchOffset, "${now.day} ${DateFormat.yMMM().format(now)}"));
```

Comment:
I don't understand what this 0 argument is doing. I've never seen a second argument in List::add, and the [reference](https://api.dart.dev/be/180791/dart-core/List/add.html) doesn't mention it either.

Reply:
Sorry, i've been mistaken with java [reference](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/List.html#add(int,E))

What i meant to do was inserting the element at first position.
This functionality is possible with [insert](https://api.dart.dev/be/180791/dart-core/List/insert.html) according to dart reference.

I'll modify the request accordingly.